### ISSUE TITLE
Fix Travis Licenses

### ISSUE THREAD

**Issue by omkarmoghe** [omkarmoghe/Pokemap#14]
All Travis builds fail due to this error:

```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> You have not accepted the license agreements of the following SDK components:
  [com.android.support.constraint:constraint-layout:1.0.0-alpha4, com.android.support.constraint:constraint-layout-solver:1.0.0-alpha4].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
```

Not sure what the right config changes are. Anyone with Travis experience can pick this up!

**Comment by Jtramey** [comment]
I should be able to fix this shortly.

**Comment by omkarmoghe** [comment]
relevant: http://stackoverflow.com/a/37829552/5734895

Will need to manually upload the licenses for the constraint layout

**Comment by Mahonster** [comment]
Was this resolved in removing the constraint layout?

**Comment by fess89** [comment]
Nope, there is another license missing:
`> You have not accepted the license agreements of the following SDK components:
  [Google Repository].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html`

Looks like we need to use that "license agreement transfer" mechanism...

**Comment by omkarmoghe** [comment]
@Mahonster nah im now getting this:

```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> You have not accepted the license agreements of the following SDK components:
  [Google Repository].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
```

Basically more license issues. gonna need to export them to travis or something

**Comment by omkarmoghe** [comment]
fixed by 4ee7084e8c2b7b2ce000dc58ad42c2f88f502d13
